src/js/components/settings/global.test.js

```diff
@@ -48,7 +48,7 @@ const preloadedState = {
 describe('GlobalSettings Component', () => {
   it('renders correctly', async () => {
     const { baseElement } = render(<Global />, { preloadedState });
-    await act(async () => jest.advanceTimersByTime(TIMEOUTS.fiveSeconds));
+    await act(async () => jest.advanceTimersByTime(TIMEOUTS.refreshDefault));
```

Comment:
is that on purpose: in `src/js/components/settings/artifactgeneration.js` we change to `TIMEOUTS.fiveSeconds` and in here to the default, why? (mainly curious)

Reply:
the `fiveSeconds` are for the initialization, to ensure all the requests for the configuration retrieval have settled to not interpret the absence of a config as a call to set the default values and then save these... in the test we have to wait the `fiveSeconds` + a delta to ensure the initialization processing is over and the `refreshDefault` is 10s - so we should be definitely past the `fiveSeconds` + delta time...

Reply:
aha! thank you.